app/models/check.rb

```diff
@@ -191,4 +191,8 @@ def early_ineligible_result?
   def gross_income_excess
     session_data.dig("early_result", "gross_income_excess")
   end
+
+  def is_early_result_type_gross_income?
```

Comment:
we can remove the 'is' prefix from the method name. The '?' at the end indicates this is a boolean and is the idiomatic ruby way to do things